### DISCUSSION TITLE
el-tour-step target 是否可以做成响应式的？

### DISCUSSION THREAD

**Discussion by wilsonlv** [element-plus/element-plus#17167] [Q&A]
当step进行到某一步是，通过onchange事件，来控制某个组件来显示出来，这个组件正是el-tour-step的target所指向的组件，但是此时渲染会晚于的el-tour-step的target的赋值，导致该el-tour-step只会出现在屏幕中间，希望可以将target做成响应式的，或者是增加一个beforeChange事件，来提前渲染组件

**Comment by Fuphoenixes** [comment]
target支持传Function类型`() => HTMLElement`
